rs/shell.nix

```
@@ -0,0 +1,9 @@
+{ pkgs ? import <nixpkgs> {} } : pkgs.mkShell {
+	buildInputs = with pkgs; [
+		rustc
+		cargo
+		SDL2
+		
+		rustfmt
+	];
+}
```

I think you'll [want to add `libiconv`](https://github.com/rrbutani/rosettaboy/blob/ff7a671848369b35cc372cda016aafe9f5ce3e6e/flake.nix#L157) to support macOS:

```suggestion
	buildInputs = with pkgs; [
		rustc
		cargo
		SDL2
		
		rustfmt
	] ++ (pkgs.lib.optional pkgs.targetPlatform.isMacOS pkgs.libiconv);
}
```